public/pages/Snapshots/containers/Snapshots/Snapshots.tsx

```diff
@@ -215,6 +218,28 @@ export default class Snapshots extends Component<SnapshotsProps, SnapshotsState>
     }
   };
 
+  restoreSnapshot = async (snapshotId: string, repository: string, options: object) => {
+    try {
+      const { snapshotManagementService } = this.props;
+      const response = await snapshotManagementService.restoreSnapshot(snapshotId, repository, options);
+      if (response.ok) {
+        this.context.notifications.toasts.addSuccess(`Restored snapshot ${snapshotId} to repository ${repository}.`);
+      } else {
+        this.context.notifications.toasts.addDanger(response.error);
```

Comment:
What are your thoughts on adding some generic error text to this toast as well; similar to what you have on line 231?

Reply:
Currently, a wall of text shows up.  Would it be acceptable to replace the actual error with this generic message, perhaps with a tip, like -"There was a problem restoring the snapshot. Please check names of existing indices in Index Management."?  Or do we need to keep the full error there?

Reply:
I would be more inclined to include the error in addition to a generic message, as "check the names of existing indices" doesn't explicitly say what steps the user could take to resolve the issue. @bowenlan-amzn you might have a bit more context into what would be more helpful in this situation though.